src/openms/source/FORMAT/XQuestResultXMLFile.cpp

```diff
@@ -134,14 +134,14 @@ namespace OpenMS
         spec_xml_file << "</spectrum>" << std::endl;
 
         // the preprocessed pair spectra are sorted by another index
-        // because some pairs do not yield any reasonable hits, the index from the spectrum matches or spectrum_indices does not address the right pair anymore
-        // use find with the pair of spectrum indices to find the correct index for the preprocessed common and cross-linked ion spectra
+        // because some pairs do not yield any resonable hits, the index from the spectrum matches or spectrum_indices does not address the right pair anymore
```

Comment:
reasonable?

Reply:
this just means that some spectra do not have any hits, that are worth reporting (I need to check the exact criteria, but I think for now I only exclude stuff without any aligned peaks) and are therefore skipped in the output. because of that, the index of the vector of results does not have a direct relation to the index of the spectrum or pair of spectra that the current result relates to.

Reply:
typoe "resonable"